README.md

```diff
@@ -27,21 +27,19 @@ getLogs()
 | 547               | 34            | SalesProcess.csv                 | 2018-09-13T13:27:50.414Z   | Europe/Berlin  |
 
 ## Filter and performance / followers statistics
-```
-filter(logName)
-```
-#### Example
-```
-discoveredModel("Incident_withImpactAttributes.csv")
+#### Example Usage
+```R
+# With empty trace filter sequence
+discoveredModel(lanaURL, lanaToken, logId)
+
+# With non-empty trace filter sequence
+discoveredModel(lanaURL, lanaToken, logId, traceFilterSequence = "trace sequence")
```

Comment:
Please add more documentation here about the different things that you receive through this call as well as listing all the default values and what they actually change. If you have questions about it, just let me know.